galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager.ivt/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>dev.galasa</groupId>
+		<artifactId>galasa-managers-cicsts-parent</artifactId>
+		<version>0.13.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>dev.galasa.cicsts.cemt.manager.ivt</artifactId>
+	<version>0.13.0-SNAPSHOT</version>
+	<packaging>bundle</packaging>
+
+	<name>Galasa CEMT Manager IVTs</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>dev.galasa</groupId>
+			<artifactId>dev.galasa</artifactId>
+			<version>0.13.0-SNAPSHOT</version>
+		</dependency>
+		<dependency>
```

Comment:
How can you have a dependency on the CEDA manager if you are built before it in the Jenkinsfile